code/network/stand_gui.cpp

```diff
@@ -170,16 +170,16 @@ void std_gen_set_text(char *str, int field_num)
 	}
 
 	// otherwise set the text
-	ctrl = GetDlgItem(Multi_gen_dialog, (int)MAKEINTRESOURCE(IDC_FIELD1));
+	ctrl = GetDlgItem(Multi_gen_dialog, (int)(size_t)MAKEINTRESOURCE(IDC_FIELD1));
```

Comment:
64 bit mingw, doesn't allow you to cast from a 64 bit pointer to a 32 bit integer. (loss of precision in pointers is a severe issue)
So I first have to cast it to 64 bit integer and only then to 32 bit.